### PR TITLE
Add doc for sending patches to GCC

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: doc/sending-gcc-patch.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+This guide explains what to do to send a GCC patch for review.
+
+All the commands are supposed to be run in the folder where you cloned GCC.
+
+```bash
+./contrib/gcc-changelog/git_check_commit.py
```
Comment (CohenArthur) [inline doc/sending-gcc-patch.md:6]: If you have multiple commits, this script can also take a rev-list as argument. I often use it like this:
`contrib/gcc-changelog/git_check_commit.py upstream/master..HEAD`

(assuming you have an upstream origin with a master branch that you branched from).

For a specific commit you can also use the commit's hash as argument:

`contrib/gcc-changelog/git_check_commit.py abdef78989`

**Review by CohenArthur — APPROVED** [review]

File: doc/sending-gcc-patch.md (hunk 1 above)
Comment (GuillaumeGomez) [reply doc/sending-gcc-patch.md:6]: Thanks! Updating the PR then.

**Comment by GuillaumeGomez** [comment]
Including suggestion from @CohenArthur

**Comment by GuillaumeGomez** [comment]
I'm ashamed to say that I completely missed this part. ^^'

**Pull request merged**